Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -891,6 +899,8 @@ target_include_directories(Skia INTERFACE
 target_compile_definitions(Skia PRIVATE
     SKIA_IMPLEMENTATION=1
 
+    SK_CODEC_DECODES_PNG
```

Comment:
So we use the skia built-in png decoder for ImageDiff?

Reply:
Yes, ImageDiff only links to skia.